writer/store_events.py

```diff
@@ -130,9 +130,9 @@ def store_event_list(data_dir, station_id, cluster, event_list):
             else:
                 logger.error("Strange event (no timestamp!), discarding event "
                              "(station: %s)" % station_id)
-        except:
```

Comment:
Ah yes. I ran into this issue myself and fixed some logging, without traceback. I believe getting a traceback is much nicer in python 3. Not sure about python 2.7, though.